src/index.js

```diff
@@ -61,7 +61,7 @@ export function sampleRUM(checkpoint, data = {}) {
     }
     const { weight, id, firstReadTime } = window.hlx.rum;
     if (window.hlx && window.hlx.rum && window.hlx.rum.isSelected) {
-      const knownProperties = ['weight', 'id', 'referer', 'checkpoint', 't', 'source', 'target', 'cwv'];
+      const knownProperties = ['weight', 'id', 'referer', 'checkpoint', 't', 'source', 'target', 'cwv', 'CLS', 'FID', 'LCP', 'INP'];
```

Comment:
```suggestion
      const knownProperties = ['weight', 'id', 'referer', 'checkpoint', 't', 'source', 'target', 'CLS', 'FID', 'LCP', 'INP'];
```

I think the `cwv` property does not exist, correct?